java/code/src/com/redhat/rhn/frontend/dto/VirtualSystemOverview.java

```diff
@@ -67,7 +68,8 @@ public VirtualSystemOverview(Tuple tuple) {
             stateLabel = getTupleValue(tuple, "state_label", String.class).orElse(null);
             vcpus = getTupleValue(tuple, "vcpus", Number.class).map(Number::longValue).orElse(0L);
             memory = getTupleValue(tuple, "memory", Number.class).map(Number::longValue).orElse(0L);
-            hostServerName = getTupleValue(tuple, "host_server_name", String.class).orElse(null);
+            virtualHostName = getTupleValue(tuple, "virtual_host_name", String.class).orElse(null);
+            virtualVmName = getTupleValue(tuple, "virtual_vm_name", String.class).orElse(null);
```

Comment:
`server_name` is setup the parent class IIRC. There is no need to add another field for it